test/programs/partial-mapped-type/main.ts

```diff
@@ -0,0 +1,4 @@
+type MyObject = Partial<{
+    required: number;
+    optional: number;
```

Comment:
Indeed, I just lazily copy / pasted the optional test. Will change that to something more meaningful later today. Any suggestions for the names ?